### DISCUSSION TITLE
Debug version for windows server is required

### DISCUSSION THREAD

**Discussion by lucaschambers** [BearWare/TeamTalk5#1954] [Ideas]
Hello, is it feasible to get a debug team talk server version for Windows? We have been experiencing numerous server crashes lately, and we are unsure if someone else is causing the crashes or how it is happening. Therefore, we require a debug version to install on our windows servers for troubleshooting
and understanding the cause of these crashes. Thank you in advance.

**Comment by beqabeqa473** [comment]
You can build version yourself from sources.

On 7/18/23, lucaschambers ***@***.***> wrote:
> Hello, is it feasible to get a debug team talk server version for Windows?
> We have been experiencing numerous server crashes lately, and we are unsure
> if someone else is causing the crashes or how it is happening. Therefore, we
> require a debug version to install on our windows servers for
> troubleshooting
> and understanding the cause of these crashes. Thank you in advance.
>
> --
> Reply to this email directly or view it on GitHub:
> https://github.com/BearWare/TeamTalk5/discussions/1954
> You are receiving this because you are subscribed to this thread.
>
> Message ID: ***@***.***>


-- 
with best regards Beqa Gozalishvili
Tell: +995593454005
Email: ***@***.***
Web: https://gozaltech.org
Skype: beqabeqa473
Telegram: https://t.me/gozaltech
facebook: https://facebook.com/gozaltech
twitter: https://twitter.com/beqabeqa473
Instagram: https://instagram.com/beqa.gozalishvili

**Reply by lucaschambers** [reply]
Unfortunately, I lack the necessary resources and knowledge to construct anything from the source code. Is there anyone who could assist in building a debug version of the server for Windows server? It is imperative that we determine the cause of the frequent crashes, as we currently lack any insight into the issue.

**Comment by bear101** [comment]
There is no debug version for Windows but please try and upgrade to the latest v5.14 beta.
